cime_config/usermods_dirs/cmip6_noresm_keyCLIM_cloud2/SourceMods/src.cam/hetfrz_classnuc_oslo.F90

```diff
@@ -0,0 +1,992 @@
+module hetfrz_classnuc_oslo
```

Comment:
Some of it should be, micro_mg specifically,  since it corrects a bug in CAM (both CESM-CAM and NorESM-CAM) 
For now ESCOMP-CAM use a different solution in which a limitation is hard-coded and not governed  by the parameterisation.
The other parts are really introductions of even more tuning parameters so may or may not be relevant. They are included  in the Forces code though.